utils/text_formatting.tsx

```diff
@@ -53,23 +191,27 @@ export function formatText(text, inputOptions) {
     const options = Object.assign({}, inputOptions);
 
     if (options.searchMatches) {
-        options.searchPatterns = options.searchMatches.map(convertSearchTermToRegex);
+        options.searchPatterns = options.searchMatches.map(
+            convertSearchTermToRegex
+        );
     } else {
-        options.searchPatterns = parseSearchTerms(options.searchTerm).map(convertSearchTermToRegex);
+        options.searchPatterns = parseSearchTerms(options.searchTerm).map(
+            convertSearchTermToRegex
+        );
     }
 
     if (options.renderer) {
         output = formatWithRenderer(output, options.renderer);
         output = doFormatText(output, options);
     } else if (!('markdown' in options) || options.markdown) {
-        // the markdown renderer will call doFormatText as necessary
+    // the markdown renderer will call doFormatText as necessary
```

Comment:
Can you fix the indentation on this line? I'm not sure why ESLint doesn't pick up the indentation of comments